rust/operator-binary/src/superset_controller.rs

```diff
@@ -437,6 +437,14 @@ fn build_rolegroup_config_map(
 
     config::add_superset_config(&mut config, authentication_config);
 
+    // TODO: this is true per default for versions 3.0.0 and up.
+    //    We deactivate it here to keep existing functionality.
+    //    However this is a security issue and should be configured properly
+    //    Issue: https://github.com/stackabletech/superset-operator/issues/416
+    //    See: https://github.com/apache/superset/pull/24262
+    //    See: https://superset.apache.org/docs/security/#content-security-policy-csp
+    config.insert("TALISMAN_ENABLED".to_string(), "False".to_string());
```

Comment:
Just to be sure: This can be overwritten by the user?
Had a hard time finding the configOverride code

Reply:
good point, did not think about that. The way it is now its not override-able afaik.
What about setting it as env var and use os.environ.get('TALISMAN_ENABLED') or sth like this?
Or crd/lib.rs compute_files?

Reply:
Maybe its also sufficient to move https://github.com/stackabletech/superset-operator/blob/d9337a70cc5f62508f157c0d65d8d309624c881f/rust/operator-binary/src/superset_controller.rs#L431 down to the end of the function.
I would prefer to not further complicate things by using env vars

Reply:
https://github.com/stackabletech/superset-operator/pull/415/commits/736716ba452c0c725bd2b888db4a510846357a5d